README.md

```diff
@@ -144,7 +144,22 @@ The next time you invoke a documentation build with the "Build Documentation"
 button in Xcode's Product menu, your custom `docc` will be used for the build.
 You can confirm that your custom `docc` is being used by opening the latest build
 log in Xcode's report navigator and expanding the "Compile documentation" step.
-  
+
+### Invoking `docc` from Swift Package Manager
+
+You can also test a locally built version of Swift-DocC using Swift Package Manager
+CLI. SwiftPM will try to read `DOCC_EXEC` environment variable value, and use
+the path you provded if it's set.
+
+  1. In your test `Package.swift`, add a dependency on `swift-docc-plugin`.
+     [`swift-docc-plugin`](https://github.com/apple-swift-docc-plugin) is a 
```

Comment:
https://github.com/apple/swift-docc-plugin

Reply:
Oh yikes. Sorry about the typo, and sorry that I've missed your comment ;( 

I cleaned up the language for brevity, and also referred to the plugin as "Swift-DocC Plugin" to be consistent with the rest of the readme.